selftests/functional/test_interrupt.py

```diff
@@ -104,7 +104,10 @@ def wait_until_no_badtest():
                         if old_psutil:
                             cmdline_list = psutil.Process(p).cmdline
                         else:
-                            cmdline_list = psutil.Process(p).cmdline()
+                            try:
+                                cmdline_list = psutil.Process(p).cmdline()
+                            except psutil.AccessDenied:
+                                continue
```

Comment:
It won't explode, since the iteration will be skipped with `continue`. But once again, a consistency problem. I made this fix at a later point and ended up not checking the end result. I'll fix it.